Html/running.html

```diff
@@ -37,122 +37,60 @@ <h1>Types of race 5k-10k-21k-42k</h1>
-            progress toward your goal, whether it's losing weight or running that 5K. <br>
-           </p>
          </ul>
-         <img src="Imgs/img5k.jpeg" alt="Run 5k" width="300" height="300"/> <br>
+         <div class="banner1">
+        </div>
       </div>
       <div id="Section-10k">
        <li><h2><strong>10k</strong></h2></li>  
          <ul>
           <li><h3><em>¿What is a 10K race?</em></h3></li>
            <p>
-            A 10k race has a distance of 10 kilometers. It is ideal for experienced runners looking for more of a challenge. <br>
-            It's the second most popular race after the half marathon and requires a level of fitness that balances strength, <br>
-            energy, and stamina. <br>
-            If you've already run a 5K and like to run longer distances, the 10K may be the ideal next step. <br>
-            Completing a 10km race is an achievement in itself, being able to be happy with your time no matter what. However, <br>
-            it is normal that you want to share with other runners, as well as with your previous best times. <br>
-            Your age, cardiovascular condition, and musculoskeletal health can influence your individual performance, <br> 
-            but the average 10K time is 50 to 70 minutes. <br>
-           </p>
-          <li><h3><em>Average time of a 10K race</em></h3></li>
-           <p>
-            Most runners who are reasonably fit and run an average of 25 to 50 km per week should finish this type of race in an <br>
-            average time of 50 to 70 minutes. <br>
-            More advanced runners usually finish in about 43 to 50 minutes. While exceptionally fit runners can average under 40 <br>
-            minutes, while more casual runners can exceed 60 minutes to cover those 10 kilometers. <br>
-           </p>
-          <li><h3><em>Other factors that affect the average time 10 K</em></h3></li>
-           <p>
-            Additionally, the average 10K time can depend on factors such as age, gender, and fitness level. <br>
-            Your musculoskeletal health also comes into play, so you need to take steps to reduce pain, avoid injury, and run with <br>
-            proper form. You must properly treat factors such as shin splints, plantar fasciitis, and runner's knee. <br>
-           </p>
+            A 10k race has a distance of 10 kilometers. It is ideal for experienced runners looking for more of a <br>
+            challenge. It's the second most popular race after the half marathon and requires a level of fitness <br>
+            that balances strength, energy, and stamina. If you've already run a 5K and like to run longer distances, <br>
+            the 10K may be the ideal next step. Completing a 10km race is an achievement in itself, being able to be <br>
+            happy with your time no matter what. However, it is normal that you want to share with other runners, as <br>
```

Comment:
this is super hard to read, remember to always open and close your tags on the same column, so it's easier to read